benchopt/tests/test_cli.py

```diff
@@ -285,6 +285,20 @@ def test_benchopt_caching(self, n_rep):
         out.check_output(r'Python-PGD\[step_size=1\]:',
                          repetition=5*n_rep+1)
 
+    @pytest.mark.parametrize('output_name, file_name', [
+        ('tmp_res', "tmp_res"), ("tmp_res", "tmp_res_1")])
```

Comment:
The test should all be indepedent, so you cannot rely on the fact that the first run will create the file.
Also, the file should be cleaned at the end of the test to avoid border effect, so please use `CaptureRunOutput` that will clean up the file afterward and call `run` twice in a single context to make sure we test the `uniquify` function.